api/v1/openapi.json

```diff
@@ -0,0 +1,1115 @@
+{
+        "swagger": "2.0",
+        "info": {
+                "title": "Cilium API",
+                "description": "Cilium",
+                "version": "unversioned"
```

Comment:
Will change this to v1beta to make it clear that this is being stabilized towards v1. I'm not sure an API can even be licensed in this regard so I've omitted that for now.